identity-server/test/IdentityServer.UnitTests/Validation/AuthorizeRequest Validation/Authorize_ProtocolValidation_Resources.cs

```diff
@@ -59,7 +56,7 @@ public Authorize_ProtocolValidation_Resources()
             _mockUserSession,
             Factory.CreateRequestObjectValidator(),
             new LicenseUsageTracker(new LicenseAccessor(new IdentityServerOptions(), NullLogger<LicenseAccessor>.Instance)),
-            TestLogger.Create<AuthorizeRequestValidator>());
+            new SanitizedLogger<AuthorizeRequestValidator>(TestLogger.Create<AuthorizeRequestValidator>()));
```

Comment:
Not sure what your code conventions are, but I would much prefer to avoid the duplicate generic type usage, and create a static method `SanitizedLogger.Wrap<T>(ILogger<T> logger)` which could then be used like this:

```suggestion
            SanitizedLogger.Wrap(TestLogger.Create<AuthorizeRequestValidator>()));
```